### ISSUE TITLE
Mechanism for adding custom stop words

### ISSUE THREAD

**Issue by rth** [FreeDiscovery/FreeDiscovery#64]
It would be useful to implement a mechanism for adding / managing custom stop words, for instance, with the REST API endpoint,
```
    POST /api/v0/stop-words/
          {'name': <name>,
           'data': <list[str]>}
    GET or DELETE  /api/v0/stop-words/<name>
```
where `<name>` is a user provided tag, than can then be used as an argument of `stop_words` during feature-extraction.

**Comment by rth** [comment]
This issue could be done in the following steps,
 * [ ] add a `_StopWordsWrapper` class to `freediscovery/stop_words.py` with the following prototype,
   ```py
    class _StopWordsWrapper(object):
        def __init__(self, cache_dir):
               [...]
        def save(self, name, stop_words, force=False):
               [..]
        def load(self, name):
              [..]
    ```
    where `cache_dir` is a path and `stop_words` is a list of strings. This class would then allow to save and retrive this list of strings with `joblib.save / load` under `$CACHE_DIR/stop_words/<name>.pkl`  and some tests to `freediscovery/tests/test_stop_words.py`
  * [ ] add the above mentionned REST API endpoints to `freediscovery/server/app.py` and `freediscovery/server/resources.py` (see other code in these files for an example) and add some unit tests to `test_server.py`. 

each of these steps could be a separate PR.

**Comment by dmytrobondarchuk** [comment]
OK. I'll try to implement this.

**Comment by rth** [comment]
Great, thanks! Feel free to make a PR even if it's work in progress so we can discuss it in more detail there..

**Comment by rth** [comment]
Fixed in #64.
